### ISSUE TITLE
Consider multi-line Python statements for start scene

### ISSUE THREAD

**Issue by Splines** [bhoov/manim-notebook#25]
With the behavior introduced in #17, via the `Start scene at cursor` command, the scene is started at the line of the cursor (or the whole scene is previewed when the cursor is at a `class` statement).

This might be problematic when one Python statement is spread over multiple lines. Manim will then fail and the user might not even notice if they don't have their terminal open.

Options to resolve this and more details are provided in [this comment](https://github.com/bhoov/manim-notebook/pull/17#issuecomment-2430558222):

1. Propagate this error to the user in a vscode notification
2. Implement logic to detect the end of the python expression on the selected line

Where the first option would probably be easier to implement. Maybe one could also just open the terminal if an error is encountered?

**Comment by VladimirFokow** [comment]
but how to detect if there was an error?

to check the ipython terminal somehow, to see if it's red?

**Comment by Splines** [comment]
I was able to get the stream of an already running terminal with [this code](https://github.com/bhoov/manim-notebook/blob/a793c69b0e573b1a732169b98a19dab3ede8e6f7/src/previewCode.ts#L75-L76).<br>(The code there is reallybad, it's just my first attempts to get anything related to the progress indicator to work).

**Comment by bhoov** [comment]
I agree with [this comment](https://github.com/bhoov/manim-notebook/pull/17#issuecomment-2431365901) that we can treat this as an issue for future development. For v0.1.0, we can assume that users are viewing a terminal and can see the error.
